--- .bash_profile.acs.orig	2019-08-08 16:34:58.000000000 +0200
+++ .bash_profile.acs	2019-08-08 16:34:58.000000000 +0200
@@ -258,6 +258,9 @@
 			'Fedora') 	# Fedora Core 
 					DISTRO='FEDORA'
 					REL=`cat $RH | cut -d' ' -f4` ;;
+			'CentOS') 	# CentOS
+					DISTRO='CENTOS'
+					REL=`cat $RH | cut -d' ' -f4` ;;
 		esac
 	fi
 
@@ -1338,3 +1341,7 @@
 
 alias lr="ls -ltra"
 [ -r /alma/local/bash_profile_end   ] && . /alma/local/bash_profile_end
+
+# Dummy command to ensure 0 return code even if /alma/local/bash_profile_end was not found
+# This simply redefines the previously defined variable.
+export DISTRO=$DISTRO
